tests/test_transpiler_service.py

```diff
@@ -415,3 +415,40 @@ def test_fix_ecr_ibm_strasbourg():
     )
     transpiled_circuit = cloud_transpiler_service.run(circuit)
     assert any(isinstance(gate.operation, ECRGate) for gate in list(transpiled_circuit))
+
+
+@pytest.mark.parametrize("non_valid_use_fractional_gates_param", [8, "8", "foo"])
+def test_transpile_non_valid_use_fractional_gates_param(
+    non_valid_use_fractional_gates_param,
+):
+    circuit = random_circuit(5, depth=3, seed=42)
+
+    transpiler_service = TranspilerService(
+        backend_name="ibm_brisbane",
+        optimization_level=1,
+        use_fractional_gates=non_valid_use_fractional_gates_param,
+    )
+
+    try:
+        transpiler_service.run(circuit)
+        pytest.fail("Error expected")
+    except Exception as e:
+        assert "Wrong input" in str(e)
+
+
+@pytest.mark.parametrize(
+    "valid_use_fractional_gates_param",
+    ["no", "n", "false", "f", "0", "yes", "y", "true", "t", "1"],
```

Comment:
Wow! I didn't know there were so many ways to pass `true` or `false` 🙂

Reply:
neither me, I discovered it trying to use "yes" as a non valid value 😂